qbft/spectest/comparable/types.go

```diff
@@ -2,31 +2,32 @@ package qbftcomparable
 
 import (
 	"encoding/hex"
+
 	"github.com/bloxapp/ssv-spec/qbft"
 )
 
-var RootRegister = map[string]*qbft.State{}
+var RootRegister = map[string]*qbft.RootGetter{}
```

Comment:
1. A little comment about what this is used for would be nice.
2. Why is this a global state variable? this will be an issue or limit us from running tests that use this in parallel, if possible let's not do it as global state.

Reply:
1. Done
2. Unexported variable RootRegister will no longer be accessible from variable diff to print source/target states
example: https://github.com/olegshmuelov/ssv-spec/blob/8e6311c63cdabe08f1b6be1b8f76f21153116a7e/qbft/spectest/tests/msg_processing_spectest.go#L82

Reply:
I don't understand, `RootRegister` is still global state, as it is used in testing it might be accessed and changed in parallel from different tests.

Reply:
got your point.
added PostState to MsgProcessingSpecTest as an example

Reply:
great then! Looks good to me.